package.json

```diff
@@ -117,7 +117,6 @@
     "jest": "^24.9.0",
     "jest-styled-components": "^6.3.3",
     "less": "2.7.2",
-    "lodash": "4.17.11",
```

Comment:
I don't think you should remove this line as `lodash` is _also_ used by the local dev server. `peerDependencies` ensure lodash is there in whichever project imports `ui-components`, but it doesn't enforce the presence when running `yarn start` for the local dev server in this repo.